pkg/controller/plan/migration.go

```diff
@@ -1331,6 +1341,19 @@ func (r *Migration) setDataVolumeCheckpoints(vm *plan.VMStatus) (err error) {
 	return
 }
 
+func (r *Migration) getPrecopiesMap() map[ref.Ref][]plan.Precopy {
```

Comment:
There's no need for this. You could just pass the slice of VMStatus structs instead of converting it into a map first.

Reply:
thanks, done